src/EnergyPlus/WaterCoils.cc

```diff
@@ -2363,6 +2369,12 @@ void SizeWaterCoil(EnergyPlusData &state, int const CoilNum)
                 TempSize = AutoSize; // get the autosized air volume flow rate for use in other calculations
             }
 
+            if (state.dataWaterCoils->WaterCoil(CoilNum).WaterCoilModel == iCoilModel::CoolingDetailed) { // Coil:Cooling:Water:DetailedGeometry
+                TempSize = AutoSize; // get the autosized air volume flow rate for use in other calculations
+                if (state.dataWaterCoils->WaterCoil(CoilNum).DesAirVolFlowRate == 0.0)
+                    state.dataWaterCoils->WaterCoil(CoilNum).DesAirVolFlowRate = AutoSize;
+            }
+
```

Comment:
This is a fix for detailed water cooling coil sizing problem described in issue #8877. See PR #8878 for details.